plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/style.scss

```diff
@@ -0,0 +1,213 @@
+@import "@wordpress/base-styles/_colors.native.scss";
+
+$gutenberg-blueberry: #3858e9;
+$gutenberg-blueberry-focus: #1d35b4;
+
+.settings-payments-onboarding-modal__step#business_verification {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+	overflow: hidden;
+}
+
+.settings-payments-onboarding-modal__step-business-verification {
+	min-height: 100%;
+	display: flex;
+	flex-direction: column;
+	overflow: hidden;
+	flex: 1;
+
+	&-content {
+		overflow-y: auto;
+		display: flex;
+		flex-direction: column;
+		flex: 1;
+
+		a {
+			text-decoration: none;
+		}
+
+		.stepper {
+			&__nav {
+				position: fixed;
+				top: 0;
+				left: 0;
+				right: 0;
+				height: 80px;
+				padding-top: 8px;
+				padding-left: 8px;
+				padding-right: 8px;
+				display: grid;
+				grid-template-columns: 94px 1fr; // Sizing counted for the right padding inside 1st column (logo)
+				align-items: stretch;
+				background-color: #fff;
+				border-bottom: 1px solid $gray-300;
+				z-index: 10;
+
+				&-button {
+					color: var(--wp-components-color-accent, $gutenberg-blueberry);
+					cursor: pointer;
+					background-color: transparent;
+					border: none;
+					display: flex;
+					align-items: center;
+					padding: $gap-large;
+					font-size: 14px;
+
+					.gridicons-chevron-left {
+						fill: var(--wp-components-color-accent, $gutenberg-blueberry);
+						margin-right: 2px;
+					}
+
+					&:last-child {
+						justify-self: end;
+					}
+
+					&.hide {
+						visibility: hidden;
+					}
+				}
+
+				&-logo {
+					justify-self: center;
+					align-self: center;
+					height: 17px;
+				}
+			}
+
+			&__wrapper {
+				width: 620px;
+				max-width: 620px;
+				margin: 56px auto 0;
+				display: flex;
+				flex-direction: column;
+				align-items: center;
+				margin-bottom: $gap-large;
+
+				@media screen and ( max-width: $break-xlarge ) {
+					width: 100%;
+					padding: 0 $gap;
+					margin: $gap auto 0;
+				}
+			}
+
+			&__heading {
+				font-size: 20px;
+				line-height: 24px;
+				font-weight: 500;
+				font-family: "SF Pro Display", $default-font;
+				color: $gray-900;
+				text-align: center;
+
+				@media screen and ( max-width: $break-mobile ) {
+					text-align: left;
+					text-wrap: auto;
+				}
+			}
+
+			&__subheading {
+				font-size: 13px;
+				line-height: 20px;
+				font-weight: 400;
+				text-align: center;
+				text-wrap: wrap;
+				color: $gray-700;
+				margin: $gap-small 0 $gap-largest;
+
+				@media screen and ( max-width: $break-mobile ) {
+					text-align: left;
+				}
+			}
+
+			&__content {
+				max-width: 615px;
+				width: 100%;
+			}
+
+			&__cta {
+				display: block;
+				width: 100%;
+				height: 40px; // Matching the updated WP Component. We can remove this when we update Components version.
+				margin-top: $gap-large;
+				background: var(--wp-components-color-accent, $gutenberg-blueberry); // override the MOX CTA to use Gutenberg Blueberry.
```

Comment:
I wonder about whether we should fall back to the WP theme colors rather than using our own blueberry colours here? Wouldn't it look out of place if the merchant changed their theme?

Reply:
Great question! That’s actually how button backgrounds are currently implemented on the WooPayments side, they don’t follow the WordPress theme colors. @orcungogus, should the button colors within the modal align with the WP theme styles?